refchecker/checker/llm_checker.py

```
@@ -46,21 +48,34 @@
 """
 
 
-class Claude2Checker(CheckerBase):
-    def __init__(self) -> None:
+class LLMChecker(CheckerBase):
+    def __init__(
+        self,
+        model,
+        batch_size=16
+    ) -> None:
+
         super().__init__()
-        self.prompt_temp = CLAUDE2_CHECKING_PROMPT
-        self.prompt_temp_wq = CLAUDE2_CHECKING_PROMPT_Q
-    
+        self.prompt_temp = LLM_CHECKING_PROMPT
+        self.prompt_temp_wq = LLM_CHECKING_PROMPT_Q
+        self.batch_size = batch_size
+        if model not in ['gpt4', 'claude2']:
+            self.model = model
```

do we need to raise an error here?